conf/docker_test.config

```diff
@@ -0,0 +1,29 @@
+/*
+vim: syntax=groovy
+-*- mode: groovy;-*-
+ * -------------------------------------------------
+ *  Example Nextflow config file for Travis
+ * -------------------------------------------------
+ */
+params {
+
+  genomes {
```

Comment:
As we always expect to use this genome for this profile, you could also add `params.genome = 'docker_test'` so that the command doesn't have to use `--genome`